### PR TITLE
Add Basic CI

### PR DESCRIPTION
**Pull request by tomwilkie**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+version: 2.1
```
Comment (sh0rez) [inline .circleci/config.yml:2]: Most of our own projects, including Grafana, Loki and Tanka switched to from CircleCI to Drone. Jsonnet-Bundler also uses Drone.

No strong preference here, but might be worth to be consistent here

Comment (metalmatze) [reply .circleci/config.yml:2]: Sounds good. But also happy to go forward with this.

Comment (tomwilkie) [reply .circleci/config.yml:2]: Lets go with Circle for now, as its done.  We can move to drone in the future.

**Review by sh0rez — APPROVED** [review]
Needs rebase or cherry-pick, otherwise LGTM

**Comment by sh0rez** [comment]
Why do we vendor on this project? It is a go.mod one, which should give enough guarantees to not vendor all the time – never had a problem with this in Tanka so far

**Comment by tomwilkie** [comment]
> Why do we vendor on this project? It is a go.mod one, which should give enough guarantees to not vendor all the time – never had a problem with this in Tanka so far

We vendored previously, and I'm trying to make minimal changes.  Please open a separate issue to remove vendoring.

**Pull request merged**